src/redux/reducers/examplesOptions.ts

```diff
@@ -3,29 +3,23 @@ import {
   type ExamplesOptionsAction,
   ExamplesOptionsActionType,
 } from '../types/examplesOptions';
-import {
-  exampleFirstInitialValue,
-  exampleFirstInitialValueRu,
-  exampleFirstInitialValueEn,
-} from '../../utils/initialValues/exampleFirstInitialValue';
-import {
-  exampleSecondInitialValue,
-  exampleSecondInitialValueRu,
-  exampleSecondInitialValueEn,
-} from '../../utils/initialValues/exampleSecondInitialValue';
-import {
-  exampleThirdInitialValue,
-  exampleThirdInitialValueRu,
-  exampleThirdInitialValueEn,
-} from '../../utils/initialValues/exampleThirdInitialValue';
+import { getExampleFirstInitialValue } from '../../utils/initialValues/exampleFirstInitialValue';
+import { getExampleSecondInitialValue } from '../../utils/initialValues/exampleSecondInitialValue';
+import { getExampleThirdInitialValue } from '../../utils/initialValues/exampleThirdInitialValue';
 import { LocalStorageKey, Language } from '../../utils/types/enums';
 import { setLocalItem, getLocalItem } from '../../services/browserDataStorage/localStorage';
 import type { IChart } from '../../utils/types/api/chart';
 
 const initialState: ExamplesOptionsState = {
-  exampleFirstOptions: getLocalItem<IChart>(LocalStorageKey.ExampleFirstOptions) || exampleFirstInitialValue,
-  exampleSecondOptions: getLocalItem<IChart>(LocalStorageKey.ExampleSecondOptions) || exampleSecondInitialValue,
-  exampleThirdOptions: getLocalItem<IChart>(LocalStorageKey.ExampleThirdOptions) || exampleThirdInitialValue,
+  exampleFirstOptions:
+    getLocalItem<IChart>(LocalStorageKey.ExampleFirstOptions) ||
+    getExampleFirstInitialValue(getLocalItem<string>(LocalStorageKey.Language) || Language.Ru),
+  exampleSecondOptions:
+    getLocalItem<IChart>(LocalStorageKey.ExampleSecondOptions) ||
+    getExampleSecondInitialValue(getLocalItem<string>(LocalStorageKey.Language) || Language.Ru),
+  exampleThirdOptions:
+    getLocalItem<IChart>(LocalStorageKey.ExampleThirdOptions) ||
+    getExampleThirdInitialValue(getLocalItem<string>(LocalStorageKey.Language) || Language.Ru),
 };
```

Comment:
Let's create default language constant and use it like:
getExampleFirstInitialValue(getLocalItem<string>(LocalStorageKey.Language) || DEFAULT_LANGUAGE),